scripts/test_run.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Install subnet template inside a conda env
```

Comment:
let's use staging instead of testnet so that developers are not dependent on test TAO